webapp/channels/src/components/testUtils.tsx

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
there is already an existing testUtils file in platform, you can reference it by importing `import {wrapIntl} from '@mattermost/components/src/testUtils';`